hikari/internal/deprecation.py

```diff
@@ -49,60 +47,25 @@ def warn_deprecated(
     ----------
     obj: typing.Any
         The object that is deprecated.
+    reason: str
+        The reason for the deprecation.
 
     Other Parameters
     ----------------
-    version: typing.Optional[str]
-        If specified, the version it will be removed in.
-    alternative: typing.Optional[str]
-        If specified, the alternative to use.
```

Comment:
why were these args removed and what do these changes have to do with pdoc

Reply:
_This was explained on Discord, but I will put it here for context_

The decorator seemed to completely discard the docstring, and I couldn't get it to work, which is why I make it a function only instead.

Will try to get this to work when I have time

Reply:
What about the args though